src/ui/main_menu/button.rs

```diff
@@ -92,7 +92,13 @@ pub(super) trait UiChildBuilderExt {
 
 impl UiChildBuilderExt for ChildBuilder<'_> {
     fn spawn_main_menu_buttons(&mut self, ui_assets: &UiAssets, font: Handle<Font>) -> &mut Self {
-        for action in MAIN_MENU_BUTTON_ORDER.iter() {
+        let buttons = if cfg!(feature = "arcade") {
+            vec![MainMenuButtonActionComponent::EnterInstructions]
+        } else {
+            Vec::from(MAIN_MENU_BUTTON_ORDER)
+        };
```

Comment:
Im busy this week, so ill test this over the weekend. But I think this might not be right. To know the currently selected button, we [increment/decrement a counter](https://github.com/thetawavegame/thetawave/blob/acd61c93a237569e33ea9f921850e34a3977b486/src/ui/main_menu/button.rs#L274) whenever you press up/down and mod by ` MAIN_MENU_BUTTON_ORDER.len()`. Then we [index into `MAIN_MENU_BUTTON_ORDER`](https://github.com/thetawavegame/thetawave/blob/acd61c93a237569e33ea9f921850e34a3977b486/src/ui/main_menu/button.rs#L296) to determine the event to send/action to take. So this might lead to unexpected behavior. I need to test to know, but I think it might be better 


So _really_ you either
1. Make this generic over the ActionEventEnum+ActionEventEnum handling system+array of values that correspond to those actions
2. At compile time, set `MAIN_MENU_BUTTON_ORDER` to be a singleton array when we use the arcade feature
3. Toggle different main menu setups by placing each implementation in its own plugin and enabling/disabling those plugins `UIPlugin::build` or based in  CLI parameters by making `UIPlugin`  have a field like `simplified_start_menu: bool` or `main_menu_kind: MainMenuKindEnum` . 

Basically, if all you can do is go to the instructions screen, I dont think you even need a button.

Reply:
I chose to do option 2. Once we have more content to put on the main menu, like high scores, I think it will warrants its own plugin.